cmake/FindVILLASnode.cmake

```diff
@@ -6,10 +6,36 @@ find_library(VILLASNODE_LIBRARY
 	NAMES villas
 )
 
+find_library(VILLASCOMMON_LIBRARY
+	NAMES villas-common
+)
+
+find_library(VILLASFPGA_LIBRARY
```

Comment:
I'm confused: There is no VILLASfpga anymore. It's all merged into VILLASnode. What are you searching for here?